cmake/LLVMIRUtil.cmake

```diff
@@ -111,9 +111,25 @@ function(llvmir_attach_bc_target)
   # compile lang flags
   llvmir_extract_lang_flags(IN_LANG_FLAGS ${LINKER_LANGUAGE})
 
+  # Find all header files
+  foreach(IN_FILE ${IN_FILES})
```

Comment:
maybe combine these two `foreach` loops?

Reply:
Probably not. We need to find out all include directories to include before the second `foreach` so that they can be used.

Reply:
Just make sure you leave the comment resolution to me next time because I completely missed your replies here.

Reply:
Sorry about that. I'm new to this functionality.